pytest.ini

```diff
@@ -8,7 +8,7 @@
 
 [pytest]
 python_files = 'test_*.py' , '*_test.py' , '*_tests.py', 'TestSuite_*.py'
-norecursedirs = Python/2.7.* Python/3.7.5 BinTemp Cache SDKs JenkinsScripts cmake
+norecursedirs = Python/2.7.* Python/3.10.5 BinTemp Cache SDKs JenkinsScripts cmake
```

Comment:
You can remove the `Python/2.7.*`, BinTemp, SDKs and JenkinsScripts folders since they no longer exist in the repo